coworkings_be/utils/email_verification/get_user_from_token.py

```
@@ -0,0 +1,22 @@
+from django.contrib.auth import get_user_model
+from django.utils.encoding import force_str
+from django.utils.http import urlsafe_base64_decode
+from utils.email_verification.email_verification_token_generator import (
+    email_verification_token,
+)
+
+
+def _get_user_from_email_verification_token(uidb64, token: str):
```

It would be great, if specific exception will be raised instead of return None